projects.yaml

```diff
@@ -57,6 +57,10 @@ projects:
         - *module_bl
         - *module_hic_sam3u2c
         - records/board/sam3u2c_bl.yaml
+    sam3u2c_mkit_dk_dongle_nrf5x_if:
+        - *module_if
+        - *module_hic_sam3u2c
+        - records/board/mkit_dk_dongle_nrf5x.yaml
     sam3u2c_nrf51mkit_if:
```

Comment:
Will remove the deprecated targets, but i guess they also needs to be removed from the testing framework? So i wanted that to be in place first.